mne/viz/_3d.py

```diff
@@ -415,6 +454,19 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
         View to use. See surfer.Brain().
     colorbar : bool
         If True, display colorbar on scene.
+    limits : str | 3-tuple | dict
```

Comment:
ok, but you could still do `dict(kind='auto')`, no? Same with tuple. It could just be `dict(kind='value', lim=(fmin, fmid, fmax))` or something like that? I am not very sure if there should be two ways of doing the same thing. Of course, the documentation could be improved too. Maybe, we could link to the `mne_analyze_colormap` function using `See also`?